app/notifications/validators.py

```diff
@@ -214,3 +215,34 @@ def check_service_letter_contact_id(service_id, letter_contact_id, notification_
             message = 'letter_contact_id {} does not exist in database for service id {}'\
                 .format(letter_contact_id, service_id)
             raise BadRequestError(message=message)
+
+
+def validate_address(service, letter_data):
+    address = PostalAddress.from_personalisation(
+        letter_data,
+        allow_international_letters=(INTERNATIONAL_LETTERS in str(service.permissions)),
```

Comment:
```suggestion
        allow_international_letters=service.has_permission(INTERNATIONAL_LETTERS),
```

Reply:
service can either be a serialised json object or a db.Model type. If it's serialised there isn't a has_permissions object, if its a db.Model permissions is a resultset. So `(INTERNATIONAL_LETTERS in str(service.permissions))` works for both cases.

Reply:
Think there’s a better way (add a `has_permission` method to `SerialisedService`) but that’s a bigger refactor so best to leave out of this PR.